charts/wa-task-configuration-api/values.yaml

```diff
@@ -3,9 +3,19 @@ java:
   image: 'hmctspublic.azurecr.io/wa/task-configuration-api:latest'
   ingressHost: wa-task-configuration-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   aadIdentityName: wa
+  environment:
+    CAMUNDA_URL: "http://camunda-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal/engine-rest"
+    IDAM_URL: "https://idam-web-public.{{ .Values.global.environment }}.platform.hmcts.net"
+    S2S_URL: "http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    CCD_URL: "http://ccd-data-store-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
+    FORCE_REBUILD: 1
```

Comment:
is the FORCE_REBUILD needed ?

Reply:
I know case-api have something like that but after yesterdays talk with rpe don't think we need it.